Automattic/wp-calypso
# Stats: Restore preview button in page header

#### Proposed Changes

Address #69850.

Adds back the View Page/Post button to the header on Simple sites.

![Image](https://user-images.githubusercontent.com/40267301/200544506-c08ce9e4-67b0-4f98-baa9-f5ff3d4be354.png)

#### Testing Instructions



1. Launch the Calypso Live link.
2. Navigate to a Simple site → Stats.
3. Under Posts & pages, click on a site title to view the post summary.
4. Confirm you have a View Post button in the header. (old style)
5. Add `?flags=stats/fixed-nav-headers` to the URL.
6. Confirm you still have a View Post button in the header.

#### Pre-merge Checklist






Related to #69850
